src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ set(NATIVECOMPRESSION_SOURCES
     pal_zlib.c
 )
 
+if (CLR_CMAKE_TARGET_WIN32 AND NOT CLR_CMAKE_TARGET_ARCH_i386)
```

Comment:
```suggestion
if (CLR_CMAKE_TARGET_WIN32 AND NOT CLR_CMAKE_TARGET_ARCH_I386)
```
Casing nit

Reply:
Thanks - I built all architectures locally so it must not be case sensitive, but fine to fix this.

Reply:
Yes, cmake is case-insensitive. Our convention is to use all upper case for these.